platforms.sh

```diff
@@ -490,10 +490,6 @@ set_hpc() {
       HPCENV=supermic.hpc.lsu.edu
       HPCENVSHORT=supermic
    fi
-   if [[ ${fqdn:0:5} == "jason" ]]; then
```

Comment:
I agree getting these specific things out of `platforms.sh` is probably a good idea

Reply:
Yeah this line is entirely dead code now.